install/installer.class.php

```diff
@@ -187,10 +187,9 @@ public function createDirectories() {
       }
 
       // Create cache directory for the template engine
-      if (! file_exists(FLYVEMDM_TEMPLATE_CACHE_PATH)) {
-         if (! mkdir(FLYVEMDM_TEMPLATE_CACHE_PATH, 0770, true)) {
-            $this->migration->displayWarning("Cannot create " . FLYVEMDM_TEMPLATE_CACHE_PATH . " directory");
-         }
+      PluginFlyvemdmCommon::recursiveRmdir(FLYVEMDM_TEMPLATE_CACHE_PATH);
```

Comment:
@DIOHz0r Can you confirm this line will run when the agent is being upgraded ? If yes, I approve, if not, please make it the cache cleaning method run for upgrade too first. I trust you, once you checked this, you an merge yourself.

Reply:
Yes, I can confirm you the execution of the line. I tested it with a clean installation and upgrade of the plugin.